packages/dashboard/stories/components/Wallet/HistoryTable.tsx

```diff
@@ -0,0 +1,41 @@
+import { story } from '@masknet/storybook-shared'
+import { HistoryTableUI as C } from '../../../src/pages/Wallets/components/HistoryTable'
+import { action } from '@storybook/addon-actions'
+import { ZerionTransactionDirection } from '@masknet/web3-shared'
+
+const { meta, of } = story(C)
+
+export default meta({ title: 'Components/Wallet/History Table' })
```

Comment:
This story fails "Component must be wrapped with <Container.Provider>"

Reply:
Is this fixed?

Reply:
yes